pages/help.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import type { NextPage } from 'next'
+import Layout from '../components/layout/Layout'
+import CSVTable from '../components/CSVTable'
+import { ThemeProvider, Typography, Divider } from '@mui/material'
+import { StyledPageContainer } from '../styles/common'
+import { TextContainer } from '../pageStyles/home.styles'
+import { theme } from '../styles/theme'
+
+const Help: NextPage = () => {
+  return (
+        <Layout>
+        <ThemeProvider theme={theme}>
+          <StyledPageContainer>
+            <Typography variant="h3">Help</Typography>
+            <Typography variant="body1">
```

Comment:
To be consistent with the styling on other pages, let's add a divider below the header